packages/core/src/components/progress/_common.scss

```diff
@@ -9,3 +9,20 @@ $progress-track-color: rgba($gray1, 0.2) !default;
 $progress-head-color: rgba($gray1, 0.8) !default;
 $dark-progress-track-color: rgba($black, 0.5) !default;
 $dark-progress-head-color: $gray3 !default;
+
+$skeleton-animation-duration: $pt-transition-duration * 20 !default;
```

Comment:
will need to stylelint disable here since we're moving the keyframes directly into _skeleton.scss